client/state/action-types.js

```diff
@@ -120,6 +120,15 @@ export const JETPACK_CONNECT_SSO_VALIDATION_SUCCESS = 'JETPACK_CONNECT_SSO_VALID
 export const JETPACK_CONNECT_STORE_SESSION = 'JETPACK_CONNECT_STORE_SESSION';
 export const NOTICE_CREATE = 'NOTICE_CREATE';
 export const NOTICE_REMOVE = 'NOTICE_REMOVE';
+export const JETPACK_SYNC_START_REQUEST = 'JETPACK_SYNC_START_REQUEST';
+export const JETPACK_SYNC_START_RECEIVED = 'JETPACK_SYNC_START_RECEIVED';
+export const JETPACK_SYNC_START_SUCCESS = 'JETPACK_SYNC_START_SUCCESS';
+export const JETPACK_SYNC_START_ERROR = 'JETPACK_SYNC_START_ERROR';
+export const JETPACK_SYNC_STATUS_REQUEST = 'JETPACK_SYNC_STATUS_REQUEST';
+export const JETPACK_SYNC_STATUS_RECEIVED = 'JETPACK_SYNC_STATUS_RECEIVED';
+export const JETPACK_SYNC_STATUS_SUCCESS = 'JETPACK_SYNC_STATUS_SUCCESS';
+export const JETPACK_SYNC_STATUS_ERROR = 'JETPACK_SYNC_STATUS_ERROR';
+export const NEW_NOTICE = 'NEW_NOTICE';
```

Comment:
I think you might want to move this line above your code. Is it still necessary?